hedera-node/hedera-app/src/test/java/com/hedera/node/app/state/merkle/StateLifecyclesTest.java

```diff
@@ -162,9 +166,13 @@ private static HederaApp createApp(final Platform platform) {
     }
 
     private static SignedState loadSignedState(final String path) throws IOException {
-        final var signedPair = SignedStateFileReader.readStateFile(Paths.get(path));
+        final PlatformContext platformContext = new DefaultPlatformContext(
+                ConfigurationHolder.getInstance().get(), new NoOpMetrics(), CryptographyHolder.get());
+        final var signedPair = SignedStateFileReader.readStateFile(platformContext, Paths.get(path));
         // Because it's possible we are loading old data, we cannot check equivalence of the hash.
-        Assertions.assertNotNull(signedPair.signedState());
-        return signedPair.signedState();
+        try (var reservedSignedState = signedPair.reservedSignedState()) {
+            Assertions.assertNotNull(reservedSignedState.get());
+            return reservedSignedState.get();
+        }
```

Comment:
Curious that this test isn't running into the same issue with a released `mutableMap` in `FcHashMapFamily` like the `ServicesStateTest` was 🤔

Reply:
Curious!